### PR TITLE
INT-1825 move create issue panel to the left as a tab + flow improvements

### PR DESCRIPTION
**Pull request by hbjORbj**
- [x] flow changes: after user comes back to VSCode from CS, automatically finish github creation (this means title and body must be stored in Redux) and show a toaster saying “Github issue created”
- [x] moving create issue panel to the left panel as a transient tab

Screenshot 2023-09-08 at 15 04 45

### CODE REVIEW

File: src/selectors/selectSourceControlTabProps.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		<pre><code>${before ?? '// paste code here'}</code></pre>
+		<p><strong>2. Expected code after transformation (Desired output of codemod)</strong></p>
+		<pre><code>${expected ?? '// paste code here'}</code></pre>
+		<p><strong>3. Faulty code obtained after running the current version of the codemod (Actual output of codemod)</strong></p>
+		<pre><code>${after ?? '// paste code here'}</code></pre>
+		<h3>Additional context</h3>
+	`;
+};
+
+type SourceControlTabProps = {
```
Comment (grzpab) [inline src/selectors/selectSourceControlTabProps.ts:27]: can you make it readonly?

File: src/selectors/selectMainWebviewViewProps.ts [hunk 2]
```diff
@@ -55,6 +56,16 @@ export const selectMainWebviewViewProps = (
 		};
 	}
 
+	if (state.activeTabId === 'sourceControl') {
+		const sourceControlTabProps = selectSourceControlTabProps(state);
+
+		return {
+			activeTabId: state.activeTabId,
+			title: sourceControlTabProps?.title ?? null,
```
Comment (grzpab) [inline src/selectors/selectMainWebviewViewProps.ts:64]: I'd make it an empty string, or return `selectSourceControlTabProps` as the view props so we show what has happened in the app

**Review by grzpab — APPROVED** [review]
Approved, the minor issues can be fixed later.

File: src/selectors/selectMainWebviewViewProps.ts (hunk 2 above)
Comment (hbjORbj) [reply src/selectors/selectMainWebviewViewProps.ts]: addressed

File: src/selectors/selectSourceControlTabProps.ts (hunk 1 above)
Comment (hbjORbj) [reply src/selectors/selectSourceControlTabProps.ts]: addressed

**Pull request merged**